master/buildbot/util/eventual.py

```diff
@@ -15,9 +15,9 @@
 
 
 # copied from foolscap
-
 from twisted.internet import reactor, defer
 from twisted.python import log
+import klog
```

Comment:
Minor: It would be nice to not introduce new pep issues. My recommendation is to add an extra blank line between import statements and class declaration

Reply:
Done